contrib/storage-hbase/src/main/java/org/apache/drill/exec/store/hbase/config/HBasePersistentStore.java

```diff
@@ -25,36 +25,40 @@
 import java.util.Map.Entry;
 import java.util.NoSuchElementException;
 
-import com.google.common.collect.Iterators;
 import org.apache.drill.common.exceptions.DrillRuntimeException;
 import org.apache.drill.exec.store.sys.BasePersistentStore;
 import org.apache.drill.exec.store.sys.PersistentStoreConfig;
 import org.apache.drill.exec.store.sys.PersistentStoreMode;
 import org.apache.hadoop.hbase.client.Delete;
 import org.apache.hadoop.hbase.client.Get;
-import org.apache.hadoop.hbase.client.HTableInterface;
 import org.apache.hadoop.hbase.client.Put;
 import org.apache.hadoop.hbase.client.Result;
 import org.apache.hadoop.hbase.client.ResultScanner;
 import org.apache.hadoop.hbase.client.Scan;
+import org.apache.hadoop.hbase.client.Table;
 import org.apache.hadoop.hbase.util.Bytes;
 
+import com.google.common.collect.Iterators;
+
 public class HBasePersistentStore<V> extends BasePersistentStore<V> {
```

Comment:
Yes. I've seen two recent patches where people are having to maintain this code. However, I thought we basically tell people not to use it. (That HDFS + ZK is our preferred paradigm.) That paradigm definitely outperforms HBase since we don't have to hit the store as part of every query (since we have listeners).